src/DwitTech.AccountService.Core/Interfaces/IActivationService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Interfaces
+{
+    public interface IActivationService
+    {
+        bool SendActivationEmail(string from, string userEmail, string templatePath, string subject);
```

Comment:
the parameter names should be something like fromEmail, toEmail. Current parameters make it seem like from isnt and email and userEmail is the only email there. Same method can also be used to send email to a non-user.

Reply:
Seen. Fixing now.